azavea/pwd-waterworks-revealed
# Feature/lol/support audio

Support audio player in quest zone. Load an associated audio clip with the deck and play it automatically.

Connects #101 

To test:

 * Load a random zone, you should see a black bar at the top with a close button.
 * Close the zone using that button.
 * Ensure swiping to the beginning and end still closes the zone.
 * Open the zone in the farthest bottom right of the map.
 * This zone has an audio file I created (so no potential license issues) feel free to add your favourite song for testing purposes. (Note, my audio clip has a few seconds of dead air at the beginning so wait for it.)
 * Try to pause the audio, start it again, replay it at the end. (text should change with these events)
 * Open the zone, close it using the X. Close it by swiping off right and left. Audio should stop when the zone closes.
